src/components/views/location/LocationButton.tsx

```diff
@@ -119,9 +119,9 @@ export function textForLocation(
 ): string {
     const date = new Date(ts).toISOString();
     if (description) {
-        return `Location "${description}" ${uri} at ${date}`;
+        return `Shared location "${description}" (${uri} at ${date})`;
     } else {
-        return `Location ${uri} at ${date}`;
+        return `Shared their location (${uri} at ${date})`;
```

Comment:
this should probably be internationalised, no?

Reply:
I'd be very jarred if I send a location on Web in Polish and open it on my phone and some random english text is shown instead

Reply:
No, it's intentionally not internationalised, because the reader will be on a different client, with potentially-different locale.

Reply:
This is the text description of the event, right? I don't really feel like `Shared their location...` is a good text representation. 

I already have a PR (https://github.com/matrix-org/matrix-react-sdk/pull/7552) to fix the notifications

Reply:
This is the fallback textual description, we normally internationalise them. Like if you're sending a file without a name we say `Attachment` in your own language, not always in English.

Reply:
OK, thanks I was obviously misinformed.  Sorry for missing your change Simon, it looks much better!

Reply:
> OK, thanks I was obviously misinformed. Sorry for missing your change Simon, it looks much better!

No worries!